src/adapter.ts

```diff
@@ -111,18 +122,18 @@ export class JasmineAdapter implements TestAdapter, IDisposable {
 			children: []
 		}
 
-		const suites: {[id: string]: TestSuiteInfo} = {};
+		const suites: { [id: string]: TestSuiteInfo } = {};
 
 		await new Promise<JasmineTestSuiteInfo | undefined>(resolve => {
-			const args = [ config.configFilePath, JSON.stringify(this.log.enabled) ];
+			const args = [config.configFilePath, JSON.stringify(this.log.enabled)];
 			const childProcess = fork(
 				require.resolve('./worker/loadTests.js'),
 				args,
 				{
 					cwd: this.workspaceFolder.uri.fsPath,
 					env: config.env,
 					execPath: config.nodePath,
-					execArgv: [],
+					execArgv: ["--require", "ts-node/register"],
```

Comment:
Thanks for your commit! I think some developers don't use typescript, some developers uses another version of ts-node. If it possible, we can specify this array of arguments in config.nodeArgv and simply attach it to execArgv